_posts/2024-10-21-reporte-boletin-poder-matriz.md

```
@@ -0,0 +1,28 @@
+comments: false
+author: Erick Burgos
+---
+ ## Nueva actualización en Matriz de Riesgo
+
+En nuestra constante búsqueda por mejorar la experiencia del usuario y la gestión de riesgos, nos complace anunciar la incorporación de dos nuevos reportes en la sección de Matriz de Riesgo de Advance: el Boletín Concursal y el Poder Judicial. Estas herramientas adicionales te permitirán tomar decisiones más informadas y mejorar la supervisión de riesgos en tu organización.
+
+ ### Boletín Concursal
+
+Este reporte lo podrán encontrar en la parte inferior de la matriz de riesgo. Una vez desplegado encontrarán la información actualizada del cliente seleccionado.
+
+<p align="center">
```

Markdownlint

Element: p
Inline HTML

(MD033, no-inline-html)